packages/client/src/components/MapView.spec.tsx

```diff
@@ -42,6 +42,8 @@ const deniedPermissions : LocationPermissionResponse = {
 
 let location : MockedObject<typeof Location>
 
+const distances = [100, 250, 500, 1000, 1500, 2000]
```

Comment:
Why is this here if it is unused?

Reply:
it was used in tests that have now been removed, ill remove this